README.md

```diff
@@ -1,7 +1,63 @@
-## Build dependencies ##
+## Overview ##
+
+Libnvcrypt is a helper library to store 128 bit extra random data in the TPM
+nvram that is added to the LUKS passphrase for encryption. For this it stores a
+mapping of LUKS uuids and keyslots to random data in the TPM chip. The free
+space in the TPM nvram limits how many keyslots can be protected with
+libnvcrypt.
+
+## Build Dependencies ##
 On Fedora: trousers-devel is needed, also all packages built with libnvcrypt
 need trousers-devel.
 
+## Basic Setup ##
+
+libnvcrypt is written with the assumption that it is the only thing using the
+TPM chip. Only the TPM chip on the Lenovo Thinkpad T450s is tested so far.
+
+### Install dependencias and start trousers ###
+Debian:
+
+    apt install trousers tpm-tools
```

Comment:
Add the `#` in front of each line, just like below for Fedora. Or remove them globally, either is fine.